packages/server/providers/couchdb.js

```diff
@@ -161,7 +169,7 @@ const couchDb = (app, io, pouchOptions) => {
     remoteDb.allDocs({ include_docs: true, attachments: true })
       .then(result => {
         const messages = result.rows.reduce((messages, { doc }) => {
-          const isNotSystem = doc._id !== wargameSettings
+          const isNotSystem = doc._id !== wargameSettings || doc._id !== settings
```

Comment:
Hmm, I think the above needs to be `&&`. Both need to be false to be sure it's not the system doc